### DISCUSSION TITLE
Repos don't show any languages

### DISCUSSION THREAD

**Discussion by appleuestc** [github-linguist/linguist#6122] [Classification] [answered]
Hi, 
Here are my two repo links which do not show any languages:
https://github.com/appleuestc/DAmbrose-Token
https://github.com/appleuestc/Keeper-app-with-ICP-Backend

Please help me fix it, thanks!

**Comment by HwangTaehyun** [comment, marked as answer]
Hi! How about this? https://github.com/github/linguist/issues/5575

Looks to be a caching issue.

**Reply by appleuestc** [reply]
This works for me! I updated the repos and languages were shown immediately. Thank you!
